keps/2936-local-queue-defaulting/README.md

```diff
@@ -0,0 +1,79 @@
+# KEP-2936: LocalQueue defaulting
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This KEP introduces a defaulting LocalQueue mechanism for the jobs that don't have
+LocalQueue name label specified.
+
+## Motivation
+
+Simplify user experience with Kueue.
+
+### Goals
+
+- default LocalQueue for jobs that don't specify LocalQueue name label.
+
+### Non-Goals
+
+- create a default LocalQueue.
+
+## Proposal
+
+If the Job doesn't have label for LocalQueue in its Spec and the LocalQueue
+with name `default` is present in Job's namespace and LocalQueueDefaulting feature gate
+is enabled, the `kueue.x-k8s.io/queue-name:default` anotation will be added to the Job.
+
+The `default` LocalQueue itself should be created manually by administrator.
+
+### User Stories
+
+#### Story 1
+
+Administrator created a single `default` LocalQueue per namespace. Users doesn't need to specify
+the LocalQueue label in order for workload to be scheduled.
+
+### Notes/Constraints/Caveats (Optional)
+
+manageJobsWithoutQueueName doesn't work with this feature, since Kueue add the LocalQueue
+label.
+
+### Risks and Mitigations
+
+## Design Details
+
+Update defautling webhook for all types of job to add `kueue.x-k8s.io/queue-name:default`
+label if the label is not present and the feature gate is enabled.
+
+### Test Plan
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes
+necessary to implement this enhancement.
+
+## Alternatives
+
```

Comment:
Could you add one more alternative, "default only at the workload spec.queueName level"?

Reply:
/hold
This is still not addressed, yet.

Reply:
Updated